src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -1,65 +1,201 @@
+            </li>
+            <li class="page-item">
+                <button class="page-link">
+                    🡺
+                </button>
+            </li>
+        </ul>
+    </nav>
+</div>
 
 @code {
 
     /// <summary>
     /// List of items to be shown
     /// </summary>
-    [Parameter]
-    public required List<string> Items { get; set; } = Examples;
+    private List<AssetItem> AssetItems { get; set; } = SampleValues;
 
-    // TODO: remove the code for the initialization
-    private static readonly List<string> Examples =
+    private static readonly List<AssetItem> SampleValues =
     [
-        "Trockeneis", "Ethanol", "Natriumhydroxid", "Aceton", "Ammoniumchlorid", "Wasserstoffperoxid",
-        "Kaliumpermanganat", "Messzylinder", "Erlenmeyerkolben", "Reagenzglas"
+        new Chemical
+        {
+            Id = new Guid(), Note = "Temperatur: -78,5\u00b0 C",
+            Room = new Room { Id = new Guid(), Name = "Audimax" }, Name = "Trockeneis", Cas = "124-38-9",
+            ItemNumber = "1845", Shop = "Eisladen", Hazards = [new Hazard { Id = new Guid(), Name = "kalt", ImagePath = "" }],
+            Unit = "kg"
+        },
+        new Chemical
+        {
+            Id = new Guid(), Note = "AKA Alkohol",
+            Room = new Room { Id = new Guid(), Name = "Bistro Athene" }, Name = "Ethanol", Cas = "64-17-5",
+            ItemNumber = "1170", Shop = "Taverne", Hazards = [new Hazard { Id = new Guid(), Name = "macht süchtig", ImagePath = "" }],
+            Unit = "l"
+        },
+        new Chemical
+        {
+            Id = new Guid(), Note = "Feststoff",
+            Room = new Room { Id = new Guid(), Name = "Bosch-Hörsaal" }, Name = "Natriumhydroxid", Cas = "1310-73-2",
+            ItemNumber = "1823", Shop = "ALDI", Hazards = [new Hazard { Id = new Guid(), Name = "ätzend", ImagePath = "" }],
+            Unit = "kg"
+        },
+        new Consumable
+        {
+            Id = new Guid(), Note = "zerbrechlich", Room = new Room { Id = new Guid(), Name = "Glashaus/10" },
+            Name = "Reagenzglas", ItemNumber = "2770/30", Shop = "Mediamarkt", Manufacturer = "Glasmacher",
+            SerialNumber = "12345678910"
+        }
     ];
 
+    /// <summary>
+    /// returns type of the given item
+    /// </summary>
+    private string GetItemType(AssetItem assetItem)
+    {
+        return assetItem is Chemical ? "Chemikalie" : "Verbrauchsmaterial";
+    }
+
+    /// <summary>
+    /// returns the CAS number
+    /// </summary>
+    private string GetCas(AssetItem assetItem)
+    {
+        if (assetItem is Chemical chemical)
+        {
+            return chemical.Cas;
+        }
+
+        return "-";
+    }
+
+    /// <summary>
+    /// returns the list of hazards of the item
+    /// </summary>
+    private List<Hazard> GetHazards(AssetItem assetItem)
+    {
+        if (assetItem is Chemical chemical)
+        {
+            return chemical.Hazards;
+        }
+
+        return [];
+    }
```

Comment:
⛏️ The implementation is completely fine, but you could leverage the power of the switch statement here. Just a more elegant way IMO.
```suggestion
    private List<Hazard> GetHazards(AssetItem assetItem)
        => assetItem switch
        {
            Chemical chemical => chemical.Hazards,
            _ => []
        };
```

Reply:
The `_` means everything else, like a default in normal switch.